### DISCUSSION TITLE
Screen flickering when saving performance

### DISCUSSION THREAD

**Discussion by ChKK1963** [probonopd/MiniDexed#254] [Q&A]
When I double click the "MiniDexed / Save" menu entry, the display switches to "Save / Performance" and is flickering, waited a couple of minutes, nothing changing, still flickering.

Single-Clicking takes me back to the "MiniDexed / Save" menu.

Is this the expected behaviour?

**Comment by probonopd** [comment]
Please retry with the latest build, should be fixed. If it still is the case, try a more powerful (Ampere) power supply.
